cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,100 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
+}.each do |p|
+  package p do
+    action :install
+  end
+end
+
+bash "/tmp/oozie.war" do
+  user "root"
+  group "root"
+  code <<-EOH
+    cp /usr/hdp/2.2.0.0-2041/oozie/oozie-server/webapps/oozie.war /tmp/oozie.war
+  EOH
+  action :run
+end
+
+directory '/tmp/oozie-hbase-prep' do
+  owner 'root'
+  group 'root'
+  mode '0755'
+  action :create
+end
+
+bash 'extract-oozie-war' do
+  user "root"
+  group "root"
+  cwd "/tmp/oozie-hbase-prep"
+  code <<-EOH
+    jar -xf ../oozie.war
+  EOH
+  action :run
+end
+
+bash 'copy-hbase-jars' do
+  user "root"
+  group "root"
+  code <<-EOH
+    cp /usr/hdp/2.2.0.0-2041/hbase/lib/hbase-common-0.98.4.2.2.0.0-2041-hadoop2.jar /tmp/oozie-hbase-prep/WEB-INF/lib/hbase-common-0.98.4.2.2.0.0-2041-hadoop2.jar
```

Comment:
Can you please not use explicit version numbers in the recipe? That goes for both the hdp version and the hbase jar versions. This will break the second we merge the 2.3 changes